### ISSUE TITLE
How to filter string with spaces ? Like if I say '**** Off' it only filter first word not the word after space? 

### ISSUE THREAD

**Issue by araza5** [ramiel/profanity-light#4]
Can you please let me know how to filter whole string "XXXX OFF" with "********"?

**Comment by ramiel** [comment]
You can try to add a word

```js
filter.addWords(['xxxx\s+off']);
```

and this should work regardless of the letter casing

**Comment by ramiel** [comment]
My suggestion seems not to work. I'll be back soon

**Comment by araza5** [comment]
Thank you for your quick response. Please let me know if you find some solution. That would be great help for me.

**Comment by ramiel** [comment]
So, the way the checker is built prevents from defining a word that contains spaces. I have an idea about how to fix it but not time today to do it, sorry. I'll work on it soon

**Comment by ramiel** [comment]
This feature is available in version 1.3.0

You can define it with

```js
filter.addWords(['xxxx\\s+off']);
```

**Comment by araza5** [comment]
Thank you Fabrizio for your proactive response. Thanks for adding this feature.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
